process/smartContract/export_test.go

```diff
@@ -1,64 +1,11 @@
 package smartContract
```

Comment:
Why does this file exist?

Reply:
Not needed anymore, removed (after a small refactor).